Cargo.toml

```diff
@@ -60,6 +65,7 @@ mbed-crypto-provider = ["psa-crypto"]
 pkcs11-provider = ["pkcs11", "picky-asn1-der", "picky-asn1", "picky-asn1-x509", "psa-crypto", "rand"]
 tpm-provider = ["tss-esapi", "picky-asn1-der", "picky-asn1", "picky-asn1-x509", "hex"]
 cryptoauthlib-provider = ["rust-cryptoauthlib"]
+trusted-service-provider = ["mbed-crypto-provider", "bindgen", "prost-build", "prost"]
```

Comment:
Sorry, this is probably not the place for a full review but have we considered pre-generating the bindings + protobuf here?

Reply:
Could do, but I'm unsure how volatile both of those are, especially in the early days of the project